docs/configuration.md

```diff
@@ -769,7 +769,7 @@ Apart from these, the following properties are also available, and may be useful
   <td><code>spark.memory.offHeap.size</code></td>
   <td>0</td>
   <td>
-    The absolute amount of memory which can be used for off-heap allocation.
+    The absolute amount of memory (in terms by bytes) which can be used for off-heap allocation.
```

Comment:
just "... memory in bytes which ..." is sufficient. "terms by bytes" doesn't work. Yes it's fine.